flexmeasures/cli/tests/test_data_add.py

```diff
@@ -211,3 +211,48 @@ def test_add_reporter(app, db, setup_dummy_data, reporter_config_raw):
         )
 
         assert len(stored_report) == 95
+
+
+@pytest.mark.skip_github
+@pytest.mark.parametrize("load_type", [("INFLEXIBLE"), ("SHIFTABLE"), ("BREAKABLE")])
+def test_add_shiftable(
+    app, db, shiftable_load_power_sensor, add_market_prices, load_type
+):
+    """
+    Schedule a 4h of consumption block at a constant power of 400kW in a day using
+    the three shiftable policies: inflexible, shiftable and breakable.
+    """
+
+    from flexmeasures.cli.data_add import add_schedule_shiftable_load
+
+    epex_da = Sensor.query.filter(Sensor.name == "epex_da").one_or_none()
+
+    shiftable_load_power_sensor_id = shiftable_load_power_sensor
+
+    cli_input_params = {
+        "sensor-id": shiftable_load_power_sensor_id,
+        "start": "2015-01-02T00:00:00+01:00",
+        "duration": "PT24H",
+        "load-duration": "PT4H",
+        "load-power": "0.4MW",
+        "load-type": load_type,
+        "consumption-price-sensor": epex_da.id,
+        "forbid": '{"start" : "2015-01-02T00:00:00+01:00", "duration" : "PT2H"}',
+    }
+
+    cli_input = to_flags(cli_input_params)
+    runner = app.test_cli_runner()
+
+    # call command
+    result = runner.invoke(add_schedule_shiftable_load, cli_input)
+
+    print(result)
+
+    assert result.exit_code == 0  # run command without errors
+
+    shiftable_load_power_sensor = Sensor.query.get(shiftable_load_power_sensor_id)
+
+    schedule = shiftable_load_power_sensor.search_beliefs()
+    assert (schedule == -0.4).event_value.sum() == 4
+
```

Comment:
We should check that the first two hours have successfully been blocked and that there are 4 load hours (probably right afterwards for INFLEXIBLE and probably later for the others, and probably BREAKABLE is the one where they are not consecutive)

Reply:
If you test these outcomes already in #729  (I believe you do) diregard this. But maybe add a comment that more detailled testing is in `data/moels/planning/tests/test_shiftable_loads.py`.